buzsaki_lab_to_nwb/huszar_hippocampus_dynamics/files_documentation.ipynb

```diff
@@ -0,0 +1,774 @@
+      "File \u001b[0;32m~/development/spikeinterface/src/spikeinterface/extractors/cellexplorersortingextractor.py:54\u001b[0m, in \u001b[0;36mCellExplorerSortingExtractor.__init__\u001b[0;34m(self, spikes_matfile_path, session_info_matfile_path, sampling_frequency)\u001b[0m\n\u001b[1;32m     52\u001b[0m     session_info_matfile_path \u001b[39m=\u001b[39m folder_path \u001b[39m/\u001b[39m \u001b[39mf\u001b[39m\u001b[39m\"\u001b[39m\u001b[39m{\u001b[39;00msorting_id\u001b[39m}\u001b[39;00m\u001b[39m.sessionInfo.mat\u001b[39m\u001b[39m\"\u001b[39m\n\u001b[1;32m     53\u001b[0m session_info_matfile_path \u001b[39m=\u001b[39m Path(session_info_matfile_path)\n\u001b[0;32m---> 54\u001b[0m \u001b[39massert\u001b[39;00m (\n\u001b[1;32m     55\u001b[0m     (session_info_matfile_path)\u001b[39m.\u001b[39mis_file()\n\u001b[1;32m     56\u001b[0m ), \u001b[39mf\u001b[39m\u001b[39m\"\u001b[39m\u001b[39mNo \u001b[39m\u001b[39m{\u001b[39;00msorting_id\u001b[39m}\u001b[39;00m\u001b[39m.sessionInfo.mat file found in the folder!\u001b[39m\u001b[39m\"\u001b[39m \n\u001b[1;32m     58\u001b[0m \u001b[39mtry\u001b[39;00m:\n\u001b[1;32m     59\u001b[0m     session_info_mat \u001b[39m=\u001b[39m scipy\u001b[39m.\u001b[39mio\u001b[39m.\u001b[39mloadmat(file_name\u001b[39m=\u001b[39m\u001b[39mstr\u001b[39m(session_info_matfile_path))\n",
+      "\u001b[0;31mAssertionError\u001b[0m: No e13_16f1_210302.sessionInfo.mat file found in the folder!"
+     ]
+    }
+   ],
+   "source": [
+    "from spikeinterface.extractors import CellExplorerSortingExtractor\n",
+    "\n",
+    "file_path = session_files_path_list[0].parent / \"e13_16f1_210302.spikes.cellinfo.mat\"\n",
+    "extractor = CellExplorerSortingExtractor(file_path)"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "No `sessionInfo` for this file so the above throws an assertion. Let's see what each of the files.\n",
+    "\n",
+    "The `CellExplorerSortingExtractor` uses the `sessionInfo` to extract the sampling frequency. But that might be somehwere else. \n",
+    "\n",
+    "Importantly, the files should contain the fields `UID` and `times` in a field called `spikes`. Let's see if any of the files contain this information and if it is consistent across them\n",
+    "\n",
+    "Looking at the files below, they do seem to agree with the basic information. It should be straighforward to role a new sorting extractor for this dataset. Or use NumpySortingExtractor and then add the `sessionInfo` manually. Not sure at this point on what it would be easier. "
```

Comment:
Is the sampling frequency the only thing it tries to use the session Info for? If so we could just make it optional in the extractor and add another optional sampling frequency argument for it

Reply:
Yeah, tha's another solution, but also, the file here from where the times and the UID do seem to be different. I think that we could make that option.